src/Home.tsx

```diff
@@ -27,6 +27,13 @@ const MintContainer = styled.div``; // add your styles here
 
 const MintButton = styled(Button)``; // add your styles here
```

Comment:
I think this button should match the style of the mint now button
![Screen Shot 2021-11-03 at 11 17 43 PM](https://user-images.githubusercontent.com/6373522/140252247-924feb5b-f384-4883-804a-7d373d742af7.png)
![Screen Shot 2021-11-03 at 11 17 59 PM](https://user-images.githubusercontent.com/6373522/140252266-5a2b6233-b6bf-4276-8aa4-4cff51b6f6b5.png)
.

Reply:
@lamar-frankie not a problem, but just to clarify - You'd like for the `Select Wallet` button to be just like the `Mint` button?